Source/GUI/preferences.cpp

```diff
@@ -116,6 +126,13 @@ void PreferencesDialog::Load()
     ui->Tracks_Audio_First->setChecked(!ActiveAllTracks[Type_Audio]);
     ui->Tracks_Audio_All->setChecked(ActiveAllTracks[Type_Audio]);
 
+    if (cacheDirectoryPathString().isEmpty())
+        ui->CacheDir_None->setChecked(true);
+    else if (cacheDirectoryPathString()==defaultCacheDirectoryPathString())
+        ui->CacheDir_Default->setChecked(true);
+    else
+        ui->CacheDir_Custom->setChecked(true);
```

Comment:
Can the custom directory be relative to the source file. Like the the source is /a/b1/c.mkv and custom is ../b2, then would output be

```
/a/b1/c.mkv
/a/b2/c.qctools.mkv
```